### DISCUSSION TITLE
Dynamic Data

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#347] [Q&A] [answered]
Hi Franky,
I have 2 questions on Dynamic Data:

1) When I use the shortcode #_DYNAMICDATA inside a single event booking form, it works perfectly. However, it is being displayed as text when I put it inside a multi-booking template:
![image](https://github.com/liedekef/events-made-easy/assets/101639308/391b07a2-b315-4ddc-a636-86f18e3bd7de)

2) I have 2 sets of Dynamic Data: 1 for Adults and 1 for Children, such that when the number of seats for Children/Adults is selected, custom fields will appear respectively for Children/Adults. In your documentation, there is a "Grouping Index" which I presume is for the use case that I just mentioned. However, it is no longer available in the current version. So, now, how can I group Dynamic Data such that the fields for Children and Adults are grouped and displayed differently?

Edit: For example, I would like to have Children's seats and dynamic data in column A and Adult's seats and dynamic data in column B. Is it possible to put the shortcode #_DYNAMICDATA{A} for Children and #_DYNAMICDATA{B} for Adults, so that it does not show duplicate fields like in the screenshot below?
![image](https://github.com/liedekef/events-made-easy/assets/101639308/f0a4005f-eb53-440e-9c01-0f34ee05d239)

**Comment by liedekef** [comment, marked as answer]
Multibooking forms are meant to send the same amount of fields per event, dynamic data would void that principle so it is not allowed.
The dynamic data condition works as follows: per condition a template is shown and that template is linked to the condition. The grouping index is hidden since it was too complicated, the documentation+screenshots have been corrected :-)

**Reply by refugeindhamma** [reply]
Thanks for the reply. I will have to find another way to combine multibooking forms and dynamic data then.
